robotoff/models.py

```diff
@@ -84,9 +84,6 @@ class ProductInsight(BaseModel):
     # (or first annotator, if multiple votes were cast).
     username = peewee.TextField(index=True, null=True)
 
-    # Latent insights don't exist anymore, this field is kept here for compatibility purpose during the migration
-    latent = peewee.BooleanField(null=False, index=True, default=False)
```

Comment:
We need a migration thus ?

Reply:
Yes, we can delete the column once it is deployed 👍